spanner/tests/system/test_system.py

```diff
@@ -274,12 +274,10 @@ def test_create_database(self):
         # We want to make sure the operation completes.
         operation.result(30)  # raises on failure / timeout.
 
-        name_attr = operator.attrgetter('name')
-        expected = sorted([temp_db, self._db], key=name_attr)
-
-        databases = list(Config.INSTANCE.list_databases())
-        found = sorted(databases, key=name_attr)
-        self.assertEqual(found, expected)
+        database_ids = [
+            database.database_id
+                for database in Config.INSTANCE.list_databases()]
```

Comment:
@tseaver This 2nd indent seems "wrong"?

Reply:
I ended up wrapping because I was avoiding using `db` as the loop variable, channellng you. :)  How should it be wrapped?

Reply:
I'm not 100% but I'm pretty sure it should just be:

```python
database_ids = [
    database.database_id
    for database in Config.INSTANCE.list_databases()]
```